editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -62,6 +62,7 @@
 #include "scene/main/canvas_layer.h"
 #include "scene/main/window.h"
 #include "scene/resources/packed_scene.h"
+#include "shader_editor_plugin.h"
```

Comment:
We use "absolute" includes to make it easier to spot where it comes from (should also be moved up next to the other editor plugins, in alphabetical order). Same for the other file.
```suggestion
#include "editor/plugins/shader_editor_plugin.h"
```

Reply:
Got it! Changed both of those in a new commit